src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  users: User[];
+  addTodo: (title: string, userId: number) => void;
+}
+
+export const AddTodoForm = (props: Props) => {
+  const { users, addTodo } = props;
+
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [newTitle, setNewTitle] = useState('');
+
+  const [showErrors, setShowErrors] = useState(false);
+
+  function resetForm() {
+    setSelectedUser(0);
+    setNewTitle('');
+  }
+
+  function handleSubmit(event: React.FormEvent) {
+    event.preventDefault();
+    if (!showErrors) {
+      setShowErrors(true);
+    }
```

Comment:
why do you show errors before checking them? maybe it would be better to check first and then show?